name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/pdf/FinTechGroupBankPDFExtractor.java

```diff
@@ -1491,6 +1504,17 @@ private void addDepotStatementTransaction()
                             t.setCurrencyCode(v.get("currency"));
                             t.setNote(trim(v.get("note")));
                         })
+                        .section("note2").optional() //
+                        .match("^[\\s]+(?<note2>ZINSPILOT Auszahlung .*)$")
+                        .assign((t, v) -> {
+                            t.setNote(trim(v.get("note2")));
+                            t.setType(AccountTransaction.Type.INTEREST);
+                        })
```

Comment:
Mixing the transaction blocks (deposit/removal) with an interest block is not a good idea. This would drag on forever in the source with future changes.

Please move the "interest pilot" to the interest block, alternatively create/move a new interest block.

Reply:
I see. Changed it to the interest handling block.